egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -42,6 +43,18 @@ def GetArgs():
     num_target_group.add_argument("--ali-dir", type=str,
                                   help="alignment directory, from which we derive the num-targets")
 
+    # CNN options
```

Comment:
when you are adding multiple arguments which all correspond to one particular part of the recipe it would be better to use a prefix. I would recommend renaming the options
<pre><code>
--cnn.layer
--cnn.bottleneck-dim
--cnn.cepstral-lifter
</code></pre>

You can pass these arguments to variables you like using the <code>dest</code> parameter
e.g.

<pre><code>
 parser.add_argument("--cnn.bottleneck-dim", type=int, dest = "cnn_bottleneck_dim",
               help="Output dimension of the linear layer at the CNN output ")
</code></pre>